templates/features.html

```diff
@@ -116,16 +122,9 @@
   }
 });
 
-search.addEventListener('keyup', function(e) {
-  // Clear results.
-  if (!e.target.value && e.keyCode != 27) { // ESC
-    featureList.filter();
-    chromeMetadata.selected = null;
-  } else {
-    //if (e.keyCode == 13) {  // Enter
-      featureList.filter(e.target.value);
-    //}
-  }
+search.addEventListener('input', function(e) {
+  featureList.filter(e.target.value);
```

Comment:
`chromeMetadata.selected = null;` is because once you start filtering, having a milestone selected makes it look like you're filtering results for that milestone. Which would be a lie :)